test/land-sale/fixtures.ts

```diff
@@ -0,0 +1,80 @@
+import {ethers, deployments, getNamedAccounts} from 'hardhat';
+import fs from 'fs-extra';
+import {SaltedProofSaleLandInfo} from '../../lib/merkleTreeHelper';
+import {Wallet} from 'ethers';
+import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+export const backendAuthWallet = new ethers.Wallet(
+  '0x4242424242424242424242424242424242424242424242424242424242424242'
+);
+export const zeroAddress = '0x0000000000000000000000000000000000000000';
+
+export const signAuthMessageAs = async (
+  wallet: Wallet | SignerWithAddress,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ...args: any[]
+): Promise<string> => {
+  const hashedData = ethers.utils.solidityKeccak256(
+    [
+      'address',
+      'address',
+      'uint256',
+      'uint256',
+      'uint256',
+      'uint256',
+      'bytes32',
+      'uint256[]',
+      'bytes32[]',
+    ],
+    args
+  );
+  return wallet.signMessage(ethers.utils.arrayify(hashedData));
+};
+
+export const setupAuthValidator = deployments.createFixture(async function (
+  hre
+) {
+  await deployments.fixture(['AuthValidator']);
+  const authValidatorContract = await ethers.getContract('AuthValidator');
+  return {
+    authValidatorContract,
+    hre,
+    getNamedAccounts,
+  };
+});
+
+export const setupEstateSale = deployments.createFixture(async function (hre) {
+  await deployments.fixture(['EstateSaleWithAuth', 'AuthValidator', 'Sand']);
```

Comment:
`await deployments.fixture('EstateSaleWithAuth');` is enough since you defined the dependencies in the deploy script

```
func.dependencies = [
  'Sand_deploy',
  'Land_deploy',
  'Asset_deploy',
  'AuthValidator_deploy',
];
```

Reply:
Great, I wasn't sure how this worked, thanks!

Reply:
Fixed